sa/precertificates.go

```diff
@@ -0,0 +1,61 @@
+package sa
+
+import (
+	"crypto/x509"
+	"time"
+
+	"golang.org/x/net/context"
+
+	"github.com/letsencrypt/boulder/core"
+	corepb "github.com/letsencrypt/boulder/core/proto"
+	sapb "github.com/letsencrypt/boulder/sa/proto"
+)
+
+// AddSerail writes a
```

Comment:
Small typo and a bit of a cliff-hanger ending :laughing:  :book:

Reply:
Done :mount_fuji: